test/libsolidity/gasTests/abiv2.sol

```diff
@@ -18,7 +18,7 @@ contract C {
 //   executionCost: 1167
 //   totalCost: 1123767
 // external:
-//   a(): 530
```

Comment:
Ah, it's just the sload increase!

Reply:
Yeah, but we should have decreases due to the net gas metering in other places.

Reply:
We don't do net gas metering because it can only be done on a transaction level.